package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Did you add this by mistake?

Reply:
Yes, not entirely sure how this got here.